python/fledge/services/core/server.py

```diff
@@ -914,7 +951,7 @@ async def _config_parents(cls):
         try:
             await cls._configuration_manager.create_category("Advanced", {}, 'Advanced', True)
             await cls._configuration_manager.create_child_category("Advanced", ["SMNTR", "SCHEDULER", "LOGGING",
-                                                                                "CONFIGURATION"])
+                                                                                "CONFIGURATION","Features"])
```

Comment:
We should capitalize this for consistency with the other child categories under the Advanced parent category, which are also in uppercase.